src/components/SigninSecretKeyForm.tsx

```diff
@@ -26,55 +28,34 @@ const TempInputEl = styled.input`
   min-width: 300px;
 `;
 
-const TempErrorEl = styled.div`
-  color: #c00;
-  margin-bottom: 20px;
-`;
-
 const TempLinkButtonEl = styled.div`
   margin-bottom: 20px;
   text-decoration: underline;
   cursor: pointer;
 `;
 
-interface SigninSecretKeyFormProps {
-  onClose?: () => void;
-}
-
-export const SigninSecretKeyForm = ({ onClose }: SigninSecretKeyFormProps) => {
+export const SigninSecretKeyForm = ({ onClose }: ModalPageProps) => {
   const dispatch = useDispatch();
   const history = useHistory();
 
   const { account } = useRedux(["account"]);
-  const { status, isAuthenticated, errorMessage } = account;
+  const { status, isAuthenticated, errorString } = account;
   const [acceptedWarning, setAcceptedWarning] = useState(false);
   const [secretKey, setSecretKey] = useState("");
-  const [pageError, setPageError] = useState("");
+  const { errorMessage, setErrorMessage } = useErrorMessage(errorString, () => {
+    dispatch(resetAccountAction());
+  });
```

Comment:
I'm guessing the first arg is the Horizon error string, but even then I'm not sure, and the second is a reset link? 

Usually, if a function has more than one argument and it's not obvious what each argument is, I prefer instead to pass an object so that the parameters can be named. ("Obvious" to me means it's a convention established in other APIs, like "addEventListener" taking first an event name and second a callback, or the naming of the function makes it obvious, like an `add()` function). 

Named arguments would the function call clearer, which would make it easier for new and existing developers to understand.